City-of-Helsinki/drupal-helfi
# UHF-219: UHF-114 Remove text-color field from hero

How to test:
- Follow the instructions it here https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/16 but use drupal-helfi installation and not the one-liner.

Also check these PRs:
https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/16
https://github.com/City-of-Helsinki/drupal-hdbt/pull/26
https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/6
